OutOfSchool/OutOfSchool.ElasticsearchData/OutOfSchool.ElasticsearchData.csproj

```diff
@@ -28,6 +28,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\OutOfSchool.Common\OutOfSchool.Common.csproj" />
+    <ProjectReference Include="..\OutOfSchool.DataAccess\OutOfSchool.DataAccess.csproj" />
```

Comment:
Please move `ProviderStatus` enum to Common project.

And remove this dependency on DataAccess project.